pkg/dynamicinstrumentation/diconfig/dwarf.go

```diff
@@ -467,6 +465,7 @@ func getTypeEntryBasicInfo(typeEntry *dwarf.Entry) (typeName string, typeSize in
 	for i := range typeEntry.Field {
 		if typeEntry.Field[i].Attr == dwarf.AttrName {
 			typeName = strings.Clone(typeEntry.Field[i].Val.(string))
+			fmt.Println("TYPE NAME", typeName)
```

Comment:
Should this be behind a debug/trace log?

Reply:
It should be deleted, was a debugging print line :)